examples/lib/stream-processing.ts

```diff
@@ -120,7 +120,7 @@ const stream = queue.enumerable() // enumerable gives us a nice chainable API fo
       };
     }
   })
-  .toStream(stack, 'Stream', {
+  .toKinesis(stack, 'Stream', {
```

Comment:
Any thoughts on `toKinesis` vs `toKinesisStream`? I.e. should I include the resource type as a suffix?

* `toSQS` or `toSQSQueue`
* `toSNS` or `toSNSTopic`
* `toKinesis` or `toKinesisStream`
* `toS3DeliveryStream` or `toFirehose`
* `toGlue` or `toGlueTable` (note: it's already `toGlueTable`, so we need to change it for consistency reasons)

Reply:
it makes sense just to be a little more verbose to where it's going to, LGTM. 

`to{Service}{Resource}` is a good convention, makes it easier to reason. thinking about those services such as SageMaker that have a lot of underlying resources (although couldn't visualize a `toSageMaker{Resource}` from the top of my head)

Reply:
Yeah, I think this is a good convention. Updated to reflect that.

I'm not that happy with `toS3DeliveryStream`. Could have gone with `toFirehoseDeliveryStream`, but how to differentiate between destinations (Redshift, ElasticSearch, Splunk, etc.) Created https://github.com/sam-goodwin/punchcard/issues/31 to follow up.